src/bindings/python/pyhirte/gen/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .hirte import *
+from .consts import *
+from .job import *
```

Comment:
AFAIK, using `import *` is bad practice in python but I understand to have a generator we might need.